### ISSUE TITLE
Missing GNOME modules

### ISSUE THREAD

**Issue by A6GibKm** [hpjansson/fornalder#17]
Giving a quick look at https://apps.gnome.org/ one can find a bunch of missing core apps

- [gnome-console](https://gitlab.gnome.org/GNOME/console)
- [connections](https://gitlab.gnome.org/GNOME/connections)
- [g-t-e](https://gitlab.gnome.org/GNOME/gnome-text-editor/)
- [libadwaita](https://gitlab.gnome.org/GNOME/libadwaita/)
- [libshumate](https://gitlab.gnome.org/GNOME/libshumate/)
- [gnome-tour](https://gitlab.gnome.org/GNOME/gnome-tour)
- [calls](https://gitlab.gnome.org/GNOME/calls)

Then on developer tools, there are:

- https://gitlab.gnome.org/World/design/palette
- https://gitlab.gnome.org/World/design/emblem
- https://gitlab.gnome.org/World/design/icon-library
- https://gitlab.gnome.org/World/design/lorem
- https://gitlab.gnome.org/World/design/symbolic-preview
- https://gitlab.gnome.org/World/design/typography
- https://gitlab.gnome.org/World/design/app-icon-preview/

One has to wonder whether these should be included, but dconf-editor *is* included.

Rust Bindings: Should these be included? I mean vala and gjs are included.  There are a bunch of bindings at https://gitlab.gnome.org/World/Rust besides what can be seen at https://github.com/gtk-rs.

Python bindings at https://gitlab.gnome.org/GNOME/pygobject are also missing.

**Comment by hpjansson** [comment]
Yes, I think all of those should be included. Thanks for pointing it out!

**Comment by hpjansson** [comment]
Adding these didn't change the overall story (and I wouldn't expect it to, since the total dataset is big), but recent years were beefed up a little, especially 2020 and 2021.

Before:

![gnome-full-firstyear-authors-year-2021](https://user-images.githubusercontent.com/234488/181128758-b929c500-c26a-475c-9e3c-3fbf08b40c5b.png)

After:

![gnome-full-firstyear-authors-year-2021-2](https://user-images.githubusercontent.com/234488/181128846-d0202c8c-ae13-48f0-b83b-8e219aebbc3d.png)

**Comment by hpjansson** [comment]
The `.rs` extension still didn't make the top 15 suffixes, but I think it eventually will as activity continues (chart not shown).
